README.md

```diff
@@ -24,45 +22,49 @@ The latest stable release of the GAP system together with all currently redistri
 [downloads page](https://www.gap-system.org/Releases/index.html).
 For installation instructions see [here](https://www.gap-system.org/Download/install.html).
 
-You can compile the current development version of GAP from this repository.
-For this to work, the following dependencies need to be installed first:
-* GNU autoconf >= 2.69
+Alternatively, you can compile the latest development version of GAP. However, most
+users should instead use the latest official release instead.
+
+If you still wish to continue, you can obtain the latest GAP code from its git
+repository at
+  <https://github.com/gap-system/gap>
+In order to compile GAP, at least the following dependencies need to be installed:
+
+* a C compiler, e.g. GCC or clang
+* GNU autoconf (we recommend 2.69 or later)
+* GNU make
```

Comment:
Hm, actually official names here are GNU Autoconf, GNU Make. But then I am imagining users typing Autoconf/Make/Clang etc. and getting errors. Maybe then GCC -> gcc for consistency?

Reply:
I'd not overthink this. People need to figure package names anyway (assuming they are on a Unix with a package manager; Windows users have a problem anyway). I suggested elsewhere that we could somewhere list typical package names for some major systems. E.g.:

>  On Ubuntu or Debian, you can install all packages required by GAP via
 > `sudo apt install build-essential autoconf automake libtool libreadline-dev libgmp-dev`
> On Fedora, use ...
> On DragonFlyBSD, use
> On macOS, use ...
>  *  ... if you are using homebrew: `brew install ...`
>  * ... if you are using Fink: `fink install 
>  * ... if you are using MacPorts: `

Etc. However, I am afraid this list might get a bit longish, and it interrupts the reading flow. Hmm.. In general, a section for macOS, and another for Windows, might be helpful. I am not sure the `README.md` is the place for all this... indeed, perhaps I should remove mention of the compiler etc. again, and instead point people to `INSTALL.md` ?

[also, the list I added in the README is missing libtool]